### ISSUE TITLE
Updating to 0.20.1 results in a warning

### ISSUE THREAD

**Issue by eygraber** [Kotlin/kotlinx-atomicfu#289]
I updated from v0.20.0 to v0.20.1 and started getting a warning when building for JS IR:

```
w: duplicate library name: org.jetbrains.kotlin:kotlinx-atomicfu-runtime
```
[Here's a sample of the warning](https://github.com/eygraber/portal/actions/runs/4514910555).

I'm using Kotlin 1.8.10 and coroutines 1.6.4

**Comment by mvicsokolova** [comment]
This warning will go, when the next release of coroutines using atomicfu `0.20.1` is published. 
Unfortunately, this is a compiler warning and it can not be suppressed.

**Comment by eygraber** [comment]
Is 1.7 Beta using it?

**Comment by mvicsokolova** [comment]
No, coroutines `1.7.0-Beta` does not use the latest atomicfu release yet.

Also, this warning is not reproduced with coroutines `1.7.0-Beta` and atomicfu `0.20.1`

**Comment by eygraber** [comment]
I'm still having this issue with atomicfu 0.20.2

I cannot update to coroutines 1.7.0-Beta, and this is blocking me from being able to build unless I downgrade atomicfu to 0.20.0
